modules/special-rules/pages/hitch-x.adoc

```diff
@@ -0,0 +1,16 @@
+= Hitch (X)
+
+{blank}[TODO: Add introductory description.]
+
+---
+
+Where your unit has the {hitch} ability:
+
+* The {hitch} unit can carry as many Infantry units as we state in brackets.
+For example, the Battlefortress has {hitch-4} -- so you can embark up to four Infantry units, as we explain in the main transport rules.
+* The {hitch} unit does not gain the abilities of embarked units. For example a {hitch} unit carrying a unit with the {save} ability does not gain the {save} ability.
+
+NOTE: A {hitch} unit does not gain the abilities of embarked units.
```

Comment:
Remove the redundant line, as the note that follows covers the same point and works better than a bullet point in this case.
```suggestion

NOTE: A {hitch} unit does not gain the abilities of embarked units.
```
We could add the example to the note, but I really don't think that such an example is necessary -- the note seems very clear and unambiguous to me.
If we did though, then I'd like to reword a little: "For example, where a unit that has the {save} ability is embarked on a {hitch} unit, the {hitch} unit does not gain the {save} ability."